drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/handlers/SourceTypeSelectorTest.java

```diff
@@ -78,7 +78,7 @@ public void validateValidDMN() {
     public void addRadioButtons() {
         reset(uploadWidgetMock);
         sourceTypeSelector.addRadioButtons();
-        assertEquals(2, sourceTypeSelector.radioButtonList.size());
+        assertEquals(3, sourceTypeSelector.radioButtonList.size());
```

Comment:
@gitgabrio according this change, do I understand correctly from now on TestScenario asset will serve for testing of:
- DRL rules- old
- DMN models - old
- PMML models - **new**

?

Reply:
@jomarko 
yes, that's the plan. The backend is ready, unfortunately frontend would need other modifications that no-one can do right now, so both PRs are on "stand-by"